lib/void.fz

```diff
@@ -76,7 +76,7 @@ void(absurd void) is absurd
 
 # NYI: possible alternative definitions base on choice types
 #
-# void : choice <> is
+# void : choice () is
 #
-# void (absurd choice<>) is absurd
+# void (absurd choice ()) is absurd
```

Comment:
I think this should be `choice`, and not `choice ()` since the latter is a choice of the empty tuple.